{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/cloudsim-legacy.json"}, "html": {"href": "#!/osrf/cloudsim-legacy"}, "avatar": {"href": "data/bytebucket.org/ravatar/{e799e7f6-0caa-464a-97f8-8605a37ebfb1}ts=python"}}, "type": "repository", "name": "cloudsim-legacy", "full_name": "osrf/cloudsim-legacy", "uuid": "{e799e7f6-0caa-464a-97f8-8605a37ebfb1}"}, "links": {"attachments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/33/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/cloudsim-legacy/issues/33.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/33/watch"}, "comments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/33/comments_page=1.json"}, "html": {"href": "#!/osrf/cloudsim-legacy/issues/33/cloudsim-tests-not-working-on-jenkins"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/33/vote"}}, "reporter": {"display_name": "Carlos Ag\u00fcero", "uuid": "{da8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D"}, "html": {"href": "https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/692bf15758111acaddae4da15a47f9e5d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsCA-0.png"}}, "nickname": "caguero", "type": "user", "account_id": "557058:4ded1ddf-947e-4154-bbd1-3dba24f1bdbd"}, "title": "Cloudsim tests not working on Jenkins", "component": null, "votes": 0, "watches": 1, "content": {"raw": "There are three errors:\r\n\r\n1. In Test command: \"inside/cgi-bin/common/startup_script_builder.py\": 'inside/cgi-bin/common/startup_script_builder.py' does not exist\r\n\r\n2. In Test command:  \"cloudsimd/launchers/cloudsim.py\": \r\n   File \"cloudsimd/launchers/cloudsim.py\", line 343, in launch\r\n   File \"/var/lib/jenkins/jobs/cloudsim-cloudsim_1.2-devel-precise-amd64/workspace/cloudsim/cloudsimd/launchers/launch_utils/sshclient.py\", line 63, in upload_file\r\n     raise SshClientException('scp failed: %s'%(output))\r\n    SshClientException: scp failed: /var/www-cloudsim-auth/cloudsim.zip: No such file or directory\r\n\r\n3. In Test command: /usr/bin/python \"cloudsimd/cloudsimd_test.py\":\r\n  can't open file 'cloudsimd/cloudsimd_test.py'\r\n\r\nIt seems that two of the tests no longer exists and one of them has to be fixed.", "markup": "markdown", "html": "<p>There are three errors:</p>\n<ol>\n<li>\n<p>In Test command: \"inside/cgi-bin/common/startup_script_builder.py\": 'inside/cgi-bin/common/startup_script_builder.py' does not exist</p>\n</li>\n<li>\n<p>In Test command:  \"cloudsimd/launchers/cloudsim.py\": \n   File \"cloudsimd/launchers/cloudsim.py\", line 343, in launch\n   File \"/var/lib/jenkins/jobs/cloudsim-cloudsim_1.2-devel-precise-amd64/workspace/cloudsim/cloudsimd/launchers/launch_utils/sshclient.py\", line 63, in upload_file\n     raise SshClientException('scp failed: %s'%(output))\n    SshClientException: scp failed: /var/www-cloudsim-auth/cloudsim.zip: No such file or directory</p>\n</li>\n<li>\n<p>In Test command: /usr/bin/python \"cloudsimd/cloudsimd_test.py\":\n  can't open file 'cloudsimd/cloudsimd_test.py'</p>\n</li>\n</ol>\n<p>It seems that two of the tests no longer exists and one of them has to be fixed.</p>", "type": "rendered"}, "assignee": {"display_name": "Carlos Ag\u00fcero", "uuid": "{da8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D"}, "html": {"href": "https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/692bf15758111acaddae4da15a47f9e5d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsCA-0.png"}}, "nickname": "caguero", "type": "user", "account_id": "557058:4ded1ddf-947e-4154-bbd1-3dba24f1bdbd"}, "state": "resolved", "version": null, "edited_on": null, "created_on": "2013-02-11T20:44:58.163856+00:00", "milestone": null, "updated_on": "2013-03-21T23:00:39.501689+00:00", "type": "issue", "id": 33}